app/controllers/nonprofits_controller.rb

```diff
@@ -1,5 +1,39 @@
 class NonprofitsController < ApplicationController
+  
+  before_action :authorize_manager, only: [:edit, :update]
+  
   def show
     @nonprofit = Nonprofit.find(params[:id])
   end
+  
+  def edit
+    @nonprofit = Nonprofit.find(params[:id])
```

Comment:
@ianroberts131 Good to see the authorization protection for the `edit` and `update` actions. Consider adding a further layer of security by scoping the `find` to the `current_user`:

```rb
# Can be used on lines 10 and 14:
@nonprofit = current_user.nonprofits.find(params[:id])
```